### ISSUE TITLE
Add https://get.docker.com/TAG to (easily) download previous releases

### ISSUE THREAD

**Issue by mostolog** [docker/docker#27506]
Although it's a docker.com issue, perhaps someone in here (who considers this request interesting) can ping someone there. Otherwise, just close ;)

Scrolling trough https://github.com/docker/docker/releases I'm not able to find a way to install a specific release, cause all share the same:
> curl -fsSL https://get.docker.com/ | sh

I was wondering if adding a TAG version to URL could work:
> curl -fsSL https://get.docker.com/v1.11.0 | sh

Instead of tricks like: https://forums.docker.com/t/how-can-i-install-a-specific-version-of-the-docker-engine/1993

Regards

**Comment by thaJeztah** [comment]
Thanks for suggesting, this is a duplicate of https://github.com/docker/docker/issues/13822, and https://github.com/docker/docker/issues/9697, which was rejected

TL;DR the get.docker.com script (which is actually in this repository, so you posted in the right issue tracker; https://github.com/docker/docker/blob/master/hack/install.sh) is only meant as an "express install" to get docker setup quickly. For production/custom installs, doing a manual setup, or custom script is the recommended approach.

The docs can still be improved though, see https://github.com/docker/docker/issues/17935


I'll close this issue, but feel free to comment after I closed
